tests/unit/common/test_lambda_handler.py

```diff
@@ -0,0 +1,23 @@
+import unittest
+
+from botocore.stub import Stubber
```

Comment:
Why are you using stubber instead of https://github.com/spulec/moto specific to test lambda invocation? In my opinion its a better library to use for testing functionality that involve calls to aws apis. It's more specific than just a general response stub

Reply:
Agreed! Looks like with `Stubber` we're limited to verifying the function inputs and outputs, but not the actual state in AWS. Didn't know about `moto` but sgtm to use that instead